hoot-core/src/main/cpp/hoot/core/io/NonIdRemappingHootApiDbWriter.cpp

```diff
@@ -22,38 +22,46 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2017 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016, 2017 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
That is very strange.  That is a brand new file I created by copying an existing file.  So, there shouldn't be any prior history for it.